app/src/main/java/ch/epfl/sweng/sweng_golf/MainActivity.java

```diff
@@ -10,4 +12,11 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
-}
+
+    public void loadOfferActivity(View view){
+        Intent intent = new Intent(this, CreateOfferActivity.class);
+        intent.putExtra("username","God");
```

Comment:
Maybe put a comment like
`// TODO implement username when login effective`
so that we don't forget to replace it.